src/main/java/com/jcabi/s3/AwsOcket.java

```diff
@@ -198,6 +200,8 @@ public void write(
             );
         } catch (final AmazonServiceException ex) {
             throw new IOException(ex);
+        } catch (final InterruptedException ex) {
```

Comment:
@YamStranger since both AmazonServiceException and InterruptedException have the same treatments, you can do:

```java
} catch (final AmazonServiceException | InterruptedException ex) {
    throw new IOException(ex);
}
```